index.js

```diff
@@ -1497,6 +1498,23 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# ifElse :: (a -> Boolean) -> (a -> b) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, two unary functions and a value of any type
+  //. and applies the first function to the value if the predicate evaluates
+  //. to true; otherwise applies the second function to the value.
+  //.
+  // ```javascript
+  //. > S.ifElse(S.test(/light/), S.K('dark'), S.K('light'), 'light')
+  //. 'dark'
+  //.
+  //. > S.ifElse(S.test(/light/), S.K('dark'), S.K('light'), 'dark')
+  //. 'light'
+  //. ```
+  S.ifElse = def('ifElse', [Function, Function, Function, a], function(pred, f, g, x) {
+    return R.ifElse(pred, f, g)(x);
```

Comment:
I'd prefer:

```javascript
return pred(x) ? f(x) : g(x);
```

It's just as short, and avoids the overhead of Ramda's currying.